### ISSUE TITLE
Problem with Beacon Flood attack using Alfa AWUS036ACM

### ISSUE THREAD

**Issue by Chinakentgao** [D3Ext/WEF#32]
**Describe the bug**
The wifi I'm testing on is 2.4 and 5 Gh combined. There is no effect after turning on the beacon flood attack. I checked the wlan0 information and it shows Frequency: 2.412 GHz. Is it a problem that the 5GHz antenna is not recognized?Does it not support 802.11ac protocol? But I looked at the network card and it showed 802.11 ac. I don't know where the problem is anymore.

**Expected behavior**
There is no confusion about the names of some APs, I suspect these are 5GHz ones.

**Screenshots**
![1](https://github.com/D3Ext/WEF/assets/55339757/81c9452b-96b5-4c3e-a3fa-60ea8d5ee982)
![2](https://github.com/D3Ext/WEF/assets/55339757/c3c14f7e-e713-41e5-8358-6a010fc819ad)
![3](https://github.com/D3Ext/WEF/assets/55339757/feef1b7d-9317-4588-8b27-d988f4788378)
![4](https://github.com/D3Ext/WEF/assets/55339757/92c23466-9cc2-4a61-920b-56cde558d9c5)

**WEF version**
1.1

**System information:**
 - OS: kali linux Linux 6.1.0-kali5-amd64 Debian 6.1.12-1kali2 (2023-02-23) x86_64 GNU/Linux
 - Network card: Alfa AWUS036ACM

**Comment by D3Ext** [comment]
I don't know if this attack works over 5Ghz, however it isn't 100% effective even on 2.4 Ghz

**Comment by Chinakentgao** [comment]
yes.But I can still connect to the previous AP. I just tested and found that 5Ghz AP points cannot be scanned. But I see that Alfa AWUS036ACM supports 5Ghz, I don't know if there is something wrong with my usage.
&nbsp;


凬清沄淡
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年5月24日(星期三) 晚上11:01
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [D3Ext/WEF] Problem with Beacon Flood attack using Alfa AWUS036ACM (Issue #32)





 
Are fake APs generated?

**Comment by Chinakentgao** [comment]
After I turned off the merge function and scanned the AP, I still could not recognize the 5Ghz AP information.
![微信图片_20230525095113](https://github.com/D3Ext/WEF/assets/55339757/0658ba6d-2788-4aae-a8a9-fb06282c0297)

**Comment by Chinakentgao** [comment]
I may have found the problem, when I start the network card, I found that the wlan0 status changed to wlan0mon Frequency: 2.457 GHz. 5GHz did not appear. Does the start network card only enable the 2.4GHz module by default and not the 5GHz module? @D3Ext 
![微信图片_20230525095113](https://github.com/D3Ext/WEF/assets/55339757/34c336b5-17b9-4eaf-8ea0-627aa088b963)
